__tests__/libs/ontouml2gufo/examples.test.ts

```diff
@@ -26,6 +26,21 @@ describe('Examples', () => {
         options: {
           format: 'Turtle',
           baseIRI: 'https://alpinebits.org',
+          createInverses: true,
+        },
+      },
+      {
+        name: 'alpinebitsCustomlabel.ttl',
+        model: alpinebits,
+        options: {
+          format: 'Turtle',
+          baseIRI: 'https://alpinebits.org',
+          createInverses: true,
+          customLabels: {
+            JoK2ZeaGAqACBxS5: 'OWLPerson',
+            Organization: 'OWLOrganization',
+            'Event Plan': 'OWLEventPlan',
+          },
```

Comment:
I tested this example and got the following result:

<img width="972" alt="Schermata 2020-04-12 alle 12 45 12" src="https://user-images.githubusercontent.com/2519935/79066904-9512fe80-7cbb-11ea-888c-1f2411c8613f.png">

This means that our custom label actually applies to the element's URI and not to its label. 

Originally, this feature was meant to replace both the label and the URI, but upon seeing this case, I think we should this `customLabels` mapping should be renamed to something like `customElementMapping` so that we could aggregate both things: the selected URI and the selected labels.

```json
"customLabels": {
   "JoK2ZeaGAqACBxS5": {
      "label": { 
           "en": "Person",
           "it": "Persona",
           "pt": "Person"
      },
      "uri": "Person"
   }
}
```

Reply:
Done